konig-aws-common/src/main/java/io/konig/aws/common/GroovyAwsTearDownScriptWriter.java

```diff
@@ -80,6 +81,24 @@ private void printTableCommands() throws IOException {
 			}
 		}
 	}
+	
+	private void printAmazonBucketCommands() throws IOException {
+		File schemaDir = amazonWebService.getS3buckets();
+		if (schemaDir!=null && schemaDir.exists()) {
+			File[] fileList = schemaDir.listFiles();
+			for (File file : fileList) {
+					String path = FileUtil.relativePath(scriptFile, file);
+					print(indent);
+					print("delete AwsDeleteS3Bucket from \"");
```

Comment:
There seems to be a fundamental confusion about the data model here.  

You should NOT be introducing a new type of resource called AwsDeleteS3Bucket.  You are confusing the distinction between an action (delete) and a resource type (Aurora Table, S3 Bucket, etc.).  You should be reusing the AwsS3Bucket type here.